.github/workflows/publish.yml

```diff
@@ -72,6 +72,14 @@ jobs:
           labels: ${{ steps.aerieScheduler.outputs.labels }}
       # Publish via Gradle.
       - name: Publish Package
-        run: gradle publish
+        run: ./gradlew publish
```

Comment:
Just curious why ./gradlew and not Gradle. Still new to Gradle and my interaction with it has only been `./gradlew build `, `clean`, and creating a `task`

Reply:
Thank you for the explanation.